src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -55,7 +55,7 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
   $element: JQuery;
   $loginDialogue: JQuery;
   $restrictedDialogue: JQuery;
-  authDialogue: AuthDialogue;
+  authDialogue: AuthDialogue;  
```

Comment:
There seems to be some unwanted whitespace added at end of line in a couple of places here for some reason.